extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -438,7 +438,7 @@ export class AmpAnalytics extends AMP.BaseElement {
     request = this.expandTemplate_(request, trigger, event);
 
     // For consistency with amp-pixel we also expand any url replacements.
-    return urlReplacementsForDoc(this.win.document).expandAsync(request)
+    return urlReplacementsForDoc(this.getAmpDoc()).expandAsync(request)
```

Comment:
Used `this.element`. I am not entirely sure but `event` object in this context doesn't seem to include a `target`.